packages/round-manager/src/features/round/__tests__/RoundApplicationForm.test.tsx

```diff
@@ -62,8 +62,28 @@ describe("<RoundApplicationForm />", () => {
     (waitForSubgraphSyncTo as jest.Mock).mockResolvedValue(0);
   });
 
+  describe("when submitting form", () => {
+    it("shows headsup modal when submitted form to create a round",async () => {
+      renderWithContext(
+        <RoundApplicationForm
+          initialData={{
+            // @ts-expect-error Test file
+            program: {
+              operatorWallets: [],
+            },
+          }}
+          stepper={FormStepper}
+        />,
+      );
+      const launch = screen.getByRole("button", { name: /Launch/i });
+      fireEvent.click(launch);
+
+      expect(await screen.findByTestId("info-modal")).toBeInTheDocument();
+    });
+  });
+
   describe("when saving metadata fails", () => {
-    it("shows error modal when saving round application metadata fails", async () => {
+    it.only("shows error modal when saving round application metadata fails", async () => {
```

Comment:
This test seems to be failing when i try firing an event on the Info Modal continue Button

Reply:
resolved -- added extra `fireEvent` to click on the "Continue" button in HeadsUp modal, which is the new trigger for the create round process